### DISCUSSION TITLE
Nuxt Vite Error

### DISCUSSION THREAD

**Discussion by spencerjsmall** [posva/pinia-colada#106] [Questions] [answered]
Hi there,
First of all, super excited to be using this library. However, when I tried to install the Nuxt module into my fresh Nuxt project, 
I get the following errors when running and building the app:

```
 ERROR  Pre-transform error: Failed to resolve import "@pinia/colada" from "node_modules/.pnpm/@pinia+colada-nuxt@0.0.3_magicast@0.3.5_rollup@4.25.0/node_modules/@pinia/colada-nuxt/dist/runtime/payload-plugin.js?v=d6e8fb3d". Does the file exist?                                                                      12:18:20 PM

 ERROR  Pre-transform error: Failed to resolve import "@pinia/colada" from "node_modules/.pnpm/@pinia+colada-nuxt@0.0.3_magicast@0.3.5_rollup@4.25.0/node_modules/@pinia/colada-nuxt/dist/runtime/plugin.js?v=d6e8fb3d". Does the file exist?  
 ```
 
 I'm using pnpm and the latest version of all relevant packages:
 ```
   "dependencies": {
    "@nuxt/eslint": "^0.6.1",
    "@nuxt/ui": "https://pkg.pr.new/@nuxt/ui@v3",
    "@pinia/colada-nuxt": "^0.0.3",
    "@pinia/nuxt": "^0.7.0",
    "@stytch/vanilla-js": "^5.6.2",
    "@vue-api/nuxt": "^2.0.3",
    "@vue-flow/core": "^1.41.4",
    "@vueuse/core": "^11.2.0",
    "@vueuse/nuxt": "^11.2.0",
    "nuxt": "^3.14.159",
    "vue": "latest",
    "vue-router": "latest"
  },
  ```
  
  Any help would be greatly appreciated! Let me know if there's more context I can provide.

**Comment by posva** [comment]
a repro would help
Otherwise I sometimes try to use `pnpm dedupe`

**Reply by spencerjsmall** [reply]
@posva here's a repro with a blank nuxt project with pinia + pinia colada added as nuxt modules, everything installed @latest: https://codesandbox.io/p/devbox/4gxcyl?embed=1&file=%2Fpackage.json
try running npm run dev

**Reply by posva** [reply, marked as answer]
I see, you need to install `@pinia/colada` too. I added it to docs 🙂

**Reply by spencerjsmall** [reply]
Oh! Easy fix, thank you!
